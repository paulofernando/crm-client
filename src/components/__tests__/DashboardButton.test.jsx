import React from 'react';
import { shallow } from 'enzyme';
import DashoardButton from '../DashoardButton';

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

describe('DashoardButton', () => {

    it('renders correctly', () => {
        const wrapper = shallow(<DashoardButton to={"/test/1"} icon={"logo"} tooltip={"Tooltip test"}/>);
        expect(wrapper).toMatchSnapshot();
    });
    
});