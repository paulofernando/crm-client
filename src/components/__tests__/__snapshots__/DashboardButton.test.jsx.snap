// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashoardButton renders correctly 1`] = `ShallowWrapper {}`;
